### DISCUSSION TITLE
Node.js supported version

### DISCUSSION THREAD

**Discussion by kremik** [Koenkk/zigbee2mqtt#6799] [General]
Hello, based on [information on Zigbee2MQTT installation page](https://www.zigbee2mqtt.io/getting_started/running_zigbee2mqtt.html), Node.js version 12 is supported (or even version 10). So I wonder if it is just old information, or if you cannot use the current stable LTS version of Node.js - which is v14 at the moment. Thanks for info. Some plugins are asking for Node14, so I would like to know. And maybe also what it the latest supported npm version? 6 or 7 is also supported?

**Comment by Koenkk** [comment]
Docs were a bit outdated.

Current supported node version: 10, 12, 14 (recommended due to being current LTS) and 15
NPM version: 6 and 7
